ci/deploy.yml

```diff
@@ -80,3 +80,46 @@ msg-system deploy dev:
     - deploy
   script:
     - aws s3 cp packages/suite-data/files/message-system/config.v1.jws s3://data.trezor.io/config/develop/config.v1.jws
+
+# npm packages deploy to npm registry
+link deploy npm:
```

Comment:
Let's definitely rename this to `blockchain-link` as at first I thought you mean trezor-link.